src/lib/hooks/useSavePathToStorage.js

```diff
@@ -0,0 +1,22 @@
+import AsyncStorage from '../../lib/utils/asyncStorage'
+import { DESTINATION_PATH } from '../../lib/constants/localStorage'
+import DeepLinking from '../../lib/utils/deepLinking'
+
+export const savePathToStorage = () => {
```

Comment:
1. this isn't a React hook actually. So:
a) move it pls somewhere to the utils
b) remove 'use' prefix from its name

2. why do you use factory fn pattern here ? savePathToStorage doesn't accepts any config options to be enclosed with useSavePathToStorage. you could just declare ` const savePathToStorage = async (params, log) => { // ... ` and export it as default (or use named export)

3. `return null` is looking redundant

Reply:
refactored handleLinks to a util